projects/simplecore-ui/src/lib/button/button.ts

```diff
@@ -41,6 +49,9 @@ export class ButtonDirective implements OnInit {
       case "secondary":
         this.element.nativeElement.classList.add("core-button__background-secondary");
         break;
+      case "transparent":
+        this.renderer.addClass(this.element.nativeElement, "core-button__transparent");
+        break;
       default: // Primary
         this.element.nativeElement.classList.add("core-button__background-primary");
```

Comment:
Best practice: Use Renderer2 instead of native element in rare event of an XSS attack